### ISSUE TITLE
feature request: ORC-ID field (and further fields) for persons

### ISSUE THREAD

**Issue by Cutuchiqueno** [ManifoldScholar/manifold#1447]
The fields available to add information of persons such as authors are extremely limited. I think to be able to associate an ORC-ID to a person complies with a common standard in digital scholarly publishing.

There are also other information worth collecting from and presenting about authors such as the associated academic institution, the personal website etc.

It would be very helpful to have this as part of the user management and in conjunction with article presentation.

**Comment by zdavis** [comment]
Curious to hear what @mkgold @susanmpls @tsmyre think about this. Happy to add fields to the maker records, if we can agree on what fields are essential.

**Comment by susanmpls** [comment]
Including ORCID for an author (when an author has one) would be a great metadata addition to an author record. I think of it as an ISBN for an individual scholar. Use has been more prevalent in STEM than in HSS, but I think this identifier is getting more traction in the HSS community. Maybe @mkgold would be better informed about how much take up scholars in DH and HSS the ORCID is getting.

The data I would consider important are: name, ORCID, position & field of study (professor of anthropology, for example), institution. 

Normally we list other books they've published, but I think that could be conveyed in a short text bio, rather than as a metadata field. Not every author will have other publications.

**Comment by tsmyre** [comment]
I concur. Adding ORCID and Affiliation would also closely align with what what Cross-Ref collects for DOI registration and could be valuable also for those purposes.

I am curious about Position/Field of Study and other fields related to personal/professional websites and social media. How would, say, Email, Twitter, Facebook, Instagram, YouTube, Vimeo, GitHub/Lab, Website, etc. be useful and where/how might they be exposed?

**Comment by zdavis** [comment]
Where do you see this data rendering on the frontend of Manifold?

**Comment by tsmyre** [comment]
Yeah, that was my question too. A hover seems to transient. Would a Maker blade/block be too much?

**Comment by Cutuchiqueno** [comment]
a maker block could be nice but it also takes the risk of becoming visually too dense.

If you have several makers or contributors this does not work I think.

Another solution would be to do something similar to the way metadata is shown while reading the publication, a kind of click-initiated overlay. Also the way metadata for makers is added in the admin interface (a sliding-in `div` from the right) could work, may even less disturbing than a complete overlay.

Both options would stay consistent with the way Manifold presents similar content in other contexts.

**Comment by zdavis** [comment]
Probably good to tackle at the same time as #1504

**Comment by ja573** [comment]
I do find the author's institution very useful, but have found it difficult to normalise in a database in the past. When adding an author to a project we should be able to select the institution that person belong to at the time of publication - but when editing the person's record we should be able to add multiple institutions in order to avoid creating one 'maker' record for each author-project link.

**Comment by zdavis** [comment]
Discussed on our regular call. Decision is to include links to maker orc-id pages in the metadata.

**Comment by zdavis** [comment]
@bot accept

**Comment by zdavis** [comment]
Thank you for taking the time to open this feature request. The Manifold team reviewed this issue during our bi-weekly meeting and the consensus is that this feature makes sense and is in keeping with our overall vision for the platform. Moreover, we see this request as a viable candidate for development under our current available funding. We’re adding an “accepted” label to this request to indicate that it’s within scope and possibly within budget.

The next step is for us to estimate the work involved with this and add it to our feature backlog. Our acceptance of the issue is not a promise that it will be implemented. We will balance this request against the other accepted requests and do our best to implement it within our current available funding.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_
